app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -1115,6 +1115,8 @@ class MultipleDisposalsTriageController @Inject() (
           .map(_.fold(_ => false, _ => true))
           .getOrElse(false)
 
+        val stateYear = state.fold(_ => "", _._1.amendReturnData.fold("")(x => x.originalReturn.summary.taxYear))
```

Comment:
1. I think, `stateYear` is a correct name, please use just 'taxyear' or any meaningful name
2. We should not use this ` ""`. If you want to do the same or possible, then please get taxyear from all possible options. Otherwise, other journeys may fail. Or use different logic

Reply:
1. I have already changed the name to 'originalSubmissionYear'
2. Only 2 possible options are there. TaxYear2020 and TaxYear2021. All others will lead to error page

Reply:
No, I didn't mean possible options 2020 and 2021. I mean state have few different states (possibles) each state represents a different journey. We should get taxyear for all of them.

Reply:
What default value can we get for that one? I thought only "" is default.

Reply:
Only Amend Journey will have this value. Other journeys will have None. So falling back to Empty string is the best option